clean_project/search.py

```diff
@@ -17,27 +16,36 @@ def search(search_word, json_path):
     :return: List of files and numbers of file when search_word occurrence
```

Comment:
this method doesn't "Print all occurrence of a given word". Please fix this comment. Also, "occurrence" should be plural